update-eks-kubeconfig/update-eks-kubeconfig.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Strict Mode - http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+#URL to download the kubeconfigs
+BUCKET_URL=https://upp-kubeconfig.s3-eu-west-1.amazonaws.com
+
+# Check if the bucket is reachable
+status=$(curl -s --head -w %{http_code} $BUCKET_URL/index -o /dev/null)
+if [ $status -ne "200" ] 
+then
+  echo "S3 bucket not reachable. Check your connection"
+  exit 1
+fi
+
+# Get kubeconfigs
+for config in $(curl -s $BUCKET_URL/index); do
+curl -s $BUCKET_URL/$config > $config
+done
+
+#Merge kubeconfigs
+KUBECONFIG=$(curl -s $BUCKET_URL/index | tr "\n" ':' | sed 's/:$//g') kubectl config view --merge=true --flatten=true > /tmp/kubeconfig
+
+#Cleanup
+for config in $(curl -s $BUCKET_URL/index); do
+rm -f $config
```

Comment:
Use a mktemp to create a temp directory and then clean up the dir.

Reply:
mktemp is useful in producton servers where writing in /tmp can be a security risk. The above script is intended to be used on local computers and it is maybe good if we keep these stuff in /tmp so we can refer to them in case of failure.

Reply:
My suggestion is to use mktemp to create a temp directory in `/tmp` and then to clean it up. Not to clean up every file individually.

Reply:
Implemented temp dir with mktemp